pulp_rpm/app/viewsets/repository.py

```diff
@@ -225,10 +225,13 @@ def sync(self, request, pk):
             )
             if repository.retain_package_versions > 0:
                 raise DRFValidationError(err_msg.format("retain_package_versions"))
-        elif sync_policy == SYNC_POLICIES.MIRROR_COMPLETE:
+
+        if sync_policy == SYNC_POLICIES.MIRROR_COMPLETE:
             err_msg = "Cannot use '{}' in combination with a 'mirror_complete' sync policy."
             if repository.autopublish:
                 raise DRFValidationError(err_msg.format("autopublish"))
+            if skip_types:
```

Comment:
@dralley  why skip types is ok to use with mirror_content_only but not package_retention?

Reply:
I guess there's not an especially good reason, it's kind of arbitrary but pushes the user towards a combination that makes more logical sense. 

* With a retention policy + mirror_content_only there would be effectively no difference between mirror_content_only and additive in most cases.
* With skip_types + mirror_content_only there's still a significant behavior difference and potential benefits.